src/zenml/cli/stack_components.py

```diff
@@ -152,32 +153,64 @@ def generate_stack_component_list_command(
         A function that can be used as a `click` command.
     """
 
-    @list_options(ComponentFilter)
+    @list_options(
+        ComponentFilter,
+        default_columns=["id", "active", "name", "flavor", "owner"],
+    )
     @click.pass_context
     def list_stack_components_command(
-        ctx: click.Context, /, **kwargs: Any
+        ctx: click.Context,
+        /,
+        columns: str,
+        output_format: OutputFormat,
+        **kwargs: Any,
     ) -> None:
         """Prints a table of stack components.
 
         Args:
             ctx: The click context object
+            columns: Columns to display in output.
+            output_format: Format for output (table/json/yaml/csv/tsv).
             kwargs: Keyword arguments to filter the components.
         """
         client = Client()
         with console.status(f"Listing {component_type.plural}..."):
             kwargs["type"] = component_type
             components = client.list_stack_components(**kwargs)
-            if not components:
+            if not components.items:
                 cli_utils.declare("No components found for the given filters.")
                 return
 
-            cli_utils.print_components_table(
-                client=client,
-                component_type=component_type,
-                components=components.items,
-                show_active=not is_sorted_or_filtered(ctx),
+            show_active = not is_sorted_or_filtered(ctx)
```

Comment:
<img width="666" height="49" alt="Image" src="https://github.com/user-attachments/assets/083e1481-ddaa-4cb0-8898-dc43d2b29ed7" />

The active should be the first column like the other tables.